pkg/cmd/codespace/create.go

```diff
@@ -112,11 +113,16 @@ func (a *App) Create(ctx context.Context, opts createOptions) error {
 		return errors.New("there are no available machine types for this repository")
 	}
 
+	vscsTarget := os.Getenv("VSCS_TARGET")
+	vscsTargetUrl := os.Getenv("VSCS_TARGET_URL")
+
 	createParams := &api.CreateCodespaceParams{
 		RepositoryID:       repository.ID,
 		Branch:             branch,
 		Machine:            machine,
 		Location:           locationResult.Location,
+		VscsTarget:         vscsTarget,
+		VscsTargetUrl:      vscsTargetUrl,
```

Comment:
All letters of an initialism or acronym should use the same case: VSCSTargetURL, vscsTargetURL.